netdata.spec.in

```diff
@@ -58,6 +57,18 @@ AutoReqProv: yes
 # Conditional build:
 %bcond_without  netns    # build with netns support (cgroup-network)
 
+# No golang in el6
+%if 0%{?rhel} && 0%{?rhel} <= 6
+if [ -z "${NETDATA_DISABLE_GO+x}" ]
+  then
+	%bcond_with goplugin
+  else
+	%bcond_without goplugin
+fi
+%else
+%bcond_without goplugin
+%endif
```

Comment:
We officially do not support RHEL 6, so we don’t need special handling of it here.

Also, unless I’m mistaken, this appears to actually disable the Go plugin everywhere, which would explain why CI appears to be passing even though we never actually got the needed changes added to the package builder images (my bad on that one, I’ll jump on that later today).

Reply:
RHEL 6 condition removed